CHANGELOG-developer.asciidoc

```diff
@@ -34,6 +34,7 @@ The list below covers the major changes between 6.3.0 and master only.
 
 - Fix permissions of generated Filebeat filesets. {pull}7140[7140]
 - Collect fields from _meta/fields.yml too. {pull}8397[8397]
+- Fix issue on asset generation that could lead to different results in Windows. {pull}8464[8464]
```

Comment:
Is `CHANGELOG-developer.asciidoc` the correct doc to target? 

I notice other PRs like https://github.com/elastic/beats/pull/8147/files target `CHANGELOG.asciidoc`

Reply:
The change in this PR only affects development, it doesn't have any effect on functionality.

Reply:
Ohhhh, makes sense!